site/content/docs/howto/debugging.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Debugging Pinniped 
+description: Debugging `pinniped` configuration errors.
+description: 
+cascade:
+  layout: docs
+menu:
+  docs:
+    name: Debugging
+    weight: 900
+    parent: howtos
+---
+
+## Debugging on the Client 
+
+The `PINNIPED_DEBUG=true` environment variable can be set to enable additional CLI logging.
+
+## Debugging on the Server
+
+To adjust the log level of either the Pinniped Supervisor or the Pinniped Concierge the log level value must be updated 
+in the appropriate configmap associated with each deployment. 
+
+The `log level` options are as follows: 
+- `info` ("nice to know" information) 
+- `debug` (developer information) 
+- `trace` (timing information)
+- `all` (kitchen sink)
+
+Do not use `trace` or `all` on production systems, as credentials may get logged. When this value is left unset, 
+only warnings and errors are printed. There is no way to suppress warning and error logs.
+
+Choose an update method that corresponds with the original installation method you chose for your cluster. Consult
+[Install Supervisor]({{< ref "../howto/install-supervisor" >}}) or [Install Concierge]({{< ref "../howto/install-concierge" >}})
+for more information.
+
+### Using kapp
+
+To adjust the log level of the Pinniped Supervisor or Concierge using `kapp`, edit the `log_level` in the `values.yaml` files 
+in your local copy of the Pinniped GitHub repository. Open either `deploy/supervisor/values.yaml` or `deploy/concierge/values.yaml` 
+and edit the following line: 
+
+```yaml
+log_level: "info|debug|trace|all"
+```
+Then apply your configuration via `kapp deploy -f`.
+
+### Using kubectl
+
+To adjust the log level of the Pinniped Supervisor or Concierge using `kubectl`, find the configmap on your cluster within
+the namespace of each deployment:
+
+```bash
+# get the concierge config
+kubectl get cm pinniped-concierge-config --namespace concierge --output yaml > cm.concierge.yaml
+# get the supervisor config
+kubectl get cm pinniped-supervisor-static-config --namespace supervisor --output yaml > cm.supervisor.yaml
+```
+
+Edit the yaml files:
+
+```yaml
+# pinniped supervisor config
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: pinniped-supervisor-static-config
+  namespace: supervisor
+data:
+  pinniped.yaml: |
+    apiGroupSuffix: pinniped.dev
+    log:
+      level: "info|debug|trace|all"
+    # ...
+---
+# pinniped concierge config
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: pinniped-concierge-config
+  namespace: supervisor
+data:
+  pinniped.yaml: |
+    apiGroupSuffix: pinniped.dev
+    log:
+      level: "info|debug|trace|all"
+    # ...
+```
+
+And then apply  your configuration via `kubectl apply -f`.
```

Comment:
After applying the configuration, you must also restart the pods for the change to get picked up.

This whole section might make more sense if you split it into two sections: one for updating the log level for the Supervisor and another for updating the log level for the Concierge. The commands are slightly different, and the file that you show above would actually not have both configmaps in it if you use the commands that you show above to create the files.